manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -356,7 +356,7 @@ func (r *FybrikApplicationReconciler) reconcile(applicationContext *api.FybrikAp
 			Context: dataset.DeepCopy(),
 		}
 		if err := r.constructDataInfo(&req, applicationContext, workloadCluster); err != nil {
-			AnalyzeError(applicationContext, req.Context.DataSetID, err)
+			AnalyzeError(r.Log, applicationContext, req.Context.DataSetID, err)
 			log.Error().Err(err).Str(logging.DATASETID, req.Context.DataSetID).Msg("Error constructing data info")
```

Comment:
Redundant logging

Reply:
Thanks. I removed this log line. I hope it is okay that we lose the "Error constructing data info" message